.github/workflows/build.yaml

```diff
@@ -70,8 +70,8 @@ jobs:
       - name: Upload
         uses: actions/upload-artifact@v4
         with:
-            name: ${{ matrix.image_name }}_${{ matrix.machine }}
-            path: |
-              kas/build/tmp/deploy/images/${{ matrix.machine }}/${{ matrix.image_name }}*${{ matrix.machine }}_*.${{ matrix.image_ext }}
-              kas/build/tmp/deploy/images/${{ matrix.machine }}/${{ matrix.image_name }}*${{ matrix.machine }}_*.${{ matrix.update_ext }}
-              kas/build/tmp/deploy/images/${{ matrix.machine }}/${{ matrix.image_name }}*${{ matrix.machine }}_*.${{ matrix.manifest_ext }}
+          name: ${{ matrix.image_name }}_${{ matrix.machine }}
+          path: |
+            kas/build/tmp/deploy/images/${{ matrix.machine }}/${{ matrix.image_name }}*${{ matrix.machine }}_*.${{ matrix.image_ext }}
+            kas/build/tmp/deploy/images/${{ matrix.machine }}/${{ matrix.image_name }}*${{ matrix.machine }}_*.${{ matrix.update_ext }}
+            kas/build/tmp/deploy/images/${{ matrix.machine }}/${{ matrix.image_name }}*${{ matrix.machine }}_*.${{ matrix.manifest_ext }}
```

Comment:
Revert auto formatting changes.

(even though this change is very much needed, since the existing file was mixing space indentation between 2 and 4 spaces.)